schedview/app/prenight/prenight.py

```diff
@@ -1065,6 +1071,12 @@ def main():
 
     del prenight_app_parameters["yes"]
 
+    try:
+        show = prenight_app_parameters["show"]
+        del prenight_app_parameters["show"]
+    except KeyError:
+        show = False
```

Comment:
So .. I don't think you need this try/except, unless I'm missing something here. 
Just add a default of "False" to your add_argument definition up above, and then it will always be False unless the user sets it to be True. 
I'm also not entirely clear why you need to drop the extra keys in the prenight_app_parameters dictionary, but didn't really look in depth at what you're doing with it.

Reply:
Ok .. I went and took a look. I seems like it's so that you can keep a boundary between parameters for a piece of code that actually runs the prenight app vs. the dashboard (presumably to keep a separation between how the dashboard is configured / what port /etc. and the app itself).